Doc/reference/datamodel.rst

```diff
@@ -1657,6 +1657,18 @@ class' :attr:`~object.__dict__`.
 
    .. versionadded:: 3.6
```

Comment:
I'd be tempted to put this after the note

Reply:
Why would you put the version after the note?

Reply:
Because it matches what is done on line 1590

Reply:
Ah, I see. Thanks for the note! I will edit it in a moment.